wiki/Ranking_Criteria/en.md

```diff
@@ -86,7 +86,7 @@ Important to understand before using:
 #### Technical
 
 - **Metadata must be consistent across all difficulties of a beatmap.**
-- **[Guest difficulty](/wiki/Beatmap/Guest_difficulty) creators, storyboarders, skinners and hitsounders must be added to the tags of a beatmap.** This is to give credit where credit is due and help others identify the main contributors of any given beatmap. Usernames containing single characters separated by spaces must have the spaces replaced with underscores.
+- **[Guest difficulty](/wiki/Beatmap/Guest_difficulty) creators, storyboarders, skinners and hitsounders must be added to the tags of a beatmap.** Guest difficulty creators must be set as difficulty owners for their respective difficulties in addition to being added to tags. This is to give credit where credit is due and help others identify the main contributors of any given beatmap. Usernames in tags containing single characters separated by spaces must have the spaces replaced with underscores.
```

Comment:
If we are to keep the tag part of this, can we consider changing it to some standardised format like `guest:user_name` `skin:user_name` etc.? This can be done as a separate effort but it would help in making these tags useful in some way.

Reply:
sounds like a good idea for a possible future effort yep